Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "simplicity"
```

Comment:
to release on crates we need a different name -> https://crates.io/crates/simplicity